flow/project.py

```diff
@@ -265,8 +277,135 @@ def make_bundles(operations, size=None):
     return _make_bundles(operations, size)
 
 
+class aggregate(object):
+    """Decorator for operation functions that needs to be aggregated.
```

Comment:
It is helpful in documentation to state the default behavior of a parameter. In this case, everything is optional and the default behavior is just to pass all jobs as a single aggregate. This should be documented in the body of the documentation and the individual default behaviors mentioned with the parameters.